app/main.py

```diff
@@ -1,3 +1,50 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: (float, int)) -> None:
+        self.km = km
+
+    def __add__(self, other: (int, float, "Distance")) -> "Distance":
+        km = other if isinstance(other, (int, float)) else other.km
+        return Distance(
+            km=self.km + km
+        )
```

Comment:
No need to split into lines if it fits in one

Reply:
Done